notebooks/DrizzlePac/align_to_catalogs/align_to_catalogs.ipynb

```diff
@@ -0,0 +1,573 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Image(filename='residuals_ibwb01xxq_flc.png',width =500,height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As expected, the Gaia catalog does quite a bit better, with rms residuals less tha 0.05 pixels.  \n",
+    "\n",
+    "To apply these transformations to the image, we simply need to run TweakReg the same as before, but set the parameter `updatehdr` equal to `True`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "refcat = 'gaia.cat'\n",
+    "cw = 3.5 # Set to two times the FWHM of the PSF.\n",
+    "wcsname = 'Gaia' # Specify the WCS name for this alignment\n",
+    "\n",
+    "tweakreg.TweakReg('*flc.fits', # Pass input images\n",
+    "                  updatehdr=True, # update header with new WCS solution\n",
+    "                  imagefindcfg={'threshold': 500., 'conv_width': cw},# Detection parameters, threshold varies for different data\n",
+    "                  refcat=refcat, # Use user supplied catalog (Gaia)\n",
+    "                  interactive=False,\n",
+    "                  see2dplot=False,\n",
+    "                  shiftfile=True, # Save out shift file (so we can look at shifts later)\n",
+    "                  outshifts='Gaia_shifts.txt', # name of the shift file\n",
+    "                  wcsname=wcsname, # Give our WCS a new name\n",
+    "                  reusename=True,\n",
+    "                  sigma=2.3,\n",
+    "                  fitgeometry='general') # Use the 6 parameter fit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Drizzle the Data\n",
+    "\n",
+    "While the three sets of FLC files are now aligned, we drizzle together only the two long exposures. \n",
+    "\n",
+    "When exposures are very different lengths, drizzling them together doesn't work well when 'EXP' weighting is used. For objects that saturate in the long exposures, the problem occurs at the boundary where you transition from only short exposure to short plus long. Here the pixels getting power from long exposure pixels are only getting power from pixels whose centers are outside the ring, and thus they are weighted lower than they would be if they were getting values from both inside and outside the ring. The result is a discontinuity in the PSF radial profile and a resulting flux which is too low in those boundary pixels. For photometry of saturated objects, the short exposures should be drizzled separately from the long exposures. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "astrodrizzle.AstroDrizzle('ibwb01x[rx]q_flc.fits', \n",
+    "                          output='f606w',\n",
+    "                          preserve=False, clean=True, \n",
```

Comment:
Here and the next line, is there a particular reason for these keyword arguments to be together on one line when the others are all broken out (which in general given the complexity of `AstroDrizzle` I think is a *good* idea!)

Reply:
Agreed about the AstroDrizzle parameters. My latest commit has fixed that.